### PR TITLE
Make all manifest tags available at top level, not just some

### PR DESCRIPTION
**Pull request by davemevans**
Add (and reorder to match model.py) a bunch of missing tag objects so they are available at the top level like the ones already defined.

It'd be really cool if this could go into the next release to pypi 🤞

### CODE REVIEW

File: m3u8/__init__.py [hunk 1]
```diff
@@ -15,14 +15,16 @@
     from urllib2 import urlopen, Request, HTTPError
     from urlparse import urlparse, urljoin
 
-from m3u8.model import M3U8, Playlist, IFramePlaylist, Media, Segment, PartialSegment, PartialSegmentList, PartInformation, RenditionReport, ServerControl
+from m3u8.model import M3U8, Segment, SegmentList, PartialSegment, PartialSegmentList, Key, Playlist, IFramePlaylist, Media, MediaList, PlaylistList, Start, RenditionReport, RenditionReportList, ServerControl, Skip, PartInformation
```
Comment (mauricioabreu) [inline m3u8/__init__.py]: This line looks a bit long. Could it be improved using parentheses?

Comment (davemevans) [reply m3u8/__init__.py]: Yes, fair point. I'll make the change later.

Comment (mauricioabreu) [reply m3u8/__init__.py]: Thank you!!!

Comment (davemevans) [reply m3u8/__init__.py]: Fixed via bd764b3

**Pull request merged**